packages/react-union-scripts/scripts/webpack.config.js

```diff
@@ -118,6 +119,20 @@ const getCommonConfig = ({ outputMapper }) => ({
 	},
 });
 
+const resolveAsyncSuffix = (asyncSuffix) => {
```

Comment:
Would be great if you use R.cond and `R_.isString`, `R_.isArray` and `R_.constructRegExp`